src/main/java/plugins/nate/smp/utils/ChatUtils.java

```diff
@@ -13,4 +18,12 @@ public class ChatUtils {
     public static void sendMessage(CommandSender sender, String message) {
         sender.sendMessage(ChatColor.translateAlternateColorCodes('&', message));
     }
+
+    public static void broadcastMessage(String message, String permission) {
+        Bukkit.broadcast(ChatColor.translateAlternateColorCodes('&', message), permission);
```

Comment:
```suggestion
        Bukkit.broadcast(ChatColor.translateAlternateColorCodes('&', message), permission);
        Bukkit.getLogger().log(Level.INFO, "Broadcasting to permission node " + permission + ": " + message);

```
Add logging to the broadcast method, for traceability.